core/templates/components/forms/custom-forms-directives/thumbnail-uploader.directive.html

```diff
@@ -31,10 +31,10 @@
   thumbnail-uploader .oppia-editable-section {
     border-radius: 0.25rem;
     display: table-cell;
-    height: 100px;
+    height: 125px;
     text-align: center;
     vertical-align: middle;
-    width: 180px;
+    width: 125px;
```

Comment:
Hm, with this change, won't all uploaded images have a 1:1 aspect ratio? Did you try testing this by uploading an image? Note that different entities (topics, stories, subtopics) have different aspect ratio requirements.

Reply:
Yeah I just checked it you are right, So instead of this I thought to change the image pixels which fits perfectly fine and the aspect ratio will not get disturbed.

![Screenshot from 2021-05-05 13-13-22](https://user-images.githubusercontent.com/44300735/117111278-f7e2a680-ada4-11eb-9f99-9d2ccbe944c0.png)

![Screenshot from 2021-05-05 13-17-01](https://user-images.githubusercontent.com/44300735/117111290-fdd88780-ada4-11eb-9012-6f225ed73250.png)

But the problem I'm facing is I found the file where the ratios are given but I'm not sure on how to proceed with changes
[thumbnail-display.component.ts](https://github.com/oppia/oppia/blob/develop/core/templates/components/forms/custom-forms-directives/thumbnail-display.component.ts) and [thumbnail-display.component.html](https://github.com/oppia/oppia/blob/develop/core/templates/components/forms/custom-forms-directives/thumbnail-display.component.html)

Thanks @kevintab95